index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header class="toolbar_class">
+      <div class="logo_container">
+        <p class="logo_class">My Logo</p>
+      </div>
+      <div class="option_bar">
+        <img src="Union.png" alt="union" class="union-class">
+      </div>
+    </header>
+
+    <div class="background">
+      <div class="intro_class">
```

Comment:
- [ ] Instead of using `div` tag as the container of this section please, kindly replace it with a `<section>` tag.